contracts/proxy/ERC1967/ERC1967Utils.sol

```diff
@@ -5,7 +5,6 @@ pragma solidity ^0.8.20;
 
 import "../beacon/IBeacon.sol";
 import "../../interfaces/IERC1967.sol";
```

Comment:
Just realized this is not needed here and we should rather move it to `TransparentUpgradeableProxy`

Reply:
The goal is to re-use it as soon as possible. Should we remove + re add, or just leave it

Reply:
> The goal is to re-use it as soon as possible. 

What do you mean by "re-use it"?

> Should we remove + re add, or just leave it

The concrete suggestion would be to remove this import and add it to `TransparentUpgradeableProxy`.

Reply:
With 0.8.21 we would be able to remove the events declared here, and use the ones declared in the interface instead. When that happens, we'll need the import

Reply:
And technically `TransparentUpgradeableProxy` does not need to inherit that. This interface only contains events. By using this library, the compiler will include (in the ABI) the events declared here. In fact inheriting for the interface is something I would discourage, because it will include irrelevant events (`BeaconUpgraded` gets into `TransparentUpgradeableProxy`)

Reply:
For the sake of clarity, I moved it and pushed a commit. Feel free to revert the commit if you don't think we should do this, but for me is clear that it'd be an information issue during an audit, and we can re-add it at will once 0.8.21 is done.

I'm solving